homeassistant/components/homeassistant_connect_zbt2/update.py

```diff
@@ -156,7 +156,7 @@ async def async_setup_entry(
 class FirmwareUpdateEntity(BaseFirmwareUpdateEntity):
     """Connect ZBT-2 firmware update entity."""
 
-    bootloader_reset_type = None
+    bootloader_reset_methods = ["rts_dtr", "baudrate"]
```

Comment:
Why not use your constant?

Reply:
I've migrated these to a `StrEnum`, same as `ApplicationType`.